outputs/cloudevents/cloudevents.go

```diff
@@ -0,0 +1,67 @@
+package cloudevents
+
+import (
+	"context"
+	"errors"
+
+	ce "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/client"
+	"github.com/nsmith5/rekor-sidekick/outputs"
+)
+
+const (
+	driverName  = `cloudevents`
+	eventSource = `github.com/nsmith5/rekor-sidekick`
+	eventType   = `rekor-sidekick/event`
```

Comment:
I would rename the event type to be something more descriptive and also with a version, like:

```
eventType   = `rekor.policy.violation.v1`
```

The event is already namespaced to rekor-sidekick in the source, in cloudevents you view the "namespace" of the event in the source+event type tuple.